tests/test_config.py

```diff
@@ -0,0 +1,141 @@
+import sys
+import os
+import pathlib
+import pytest
+
+_rundir = pathlib.Path(__file__).parent
+if not str(_rundir.parent) in sys.path:
+    sys.path.insert(0, str(_rundir.parent ) )
+from util import config
+
+# A note about pytest: Things aren't completely sandboxed.  When I call
+# config.Config.get(), it selts Config._default, and that carries over
+# from one test to the next even if the call wasn't in a fixture with
+# class scope.  (The tests below are ordered with this in mind.)
+
+class TestConfig:
```

Comment:
You can see an example in my open PR.

Reply:
I have to admit that I prefer grouping together all of the tests for a class in a single test class.

But, more importantly : does it really matter?  Within one test file, yeah, we should have consistency for readability, but I don't think they all have to adhere to exactly the same way of doing it as long as they're all readable.  One thing I'd like to avoid in these reviews is reformatting for the sake of reformatting.  We aren't going to have the same style on everything, and that's OK, as long as we aren't doing things that are going to make other things harder later.

Reply:
Functionally it is the same. But I think it looks a lot more professional if we decide on one standard and conform to that. 
As for this test class thing, I don't see why the extra code and additional indentation level is needed. If you feel strongly about it or if you think it has some added value, I'd just change my tests to the same format as well.